test/release/examples/primers/distributions.chpl

```diff
@@ -240,86 +240,88 @@ verifyID(BA);
 // is replicated onto each locale. (Note: consistency among these
 // array replicands is NOT maintained automatically.)
 //
-// This replication is observable in some cases but not others,
-// as shown below. Note: this behavior may change in the future.
+// This replication is observable in some cases but not others, as
+// shown below.  In places this example uses a utility function,
```

Comment:
I didn't realize that I'd left this in such a mess.  I think I started to update it and then must've been distracted before finishing (?).  I'm going to refresh it now -- no need to put it off.